src/universal/modules/teamDashboard/components/TeamCallsToAction/TeamCallsToAction.js

```diff
@@ -39,32 +45,48 @@ const TeamCallToAction = ({history, teamId}: Props) => {
   const goToRetroLobby = () =>
     history.push(`/retro/${teamId}/`);
 
+  const itemFactory = () => {
+    const listItems = [];
+    listItems.push(
+      <MenuItem icon="arrow-circle-o-right" label="Start Action Meeting" onClick={goToMeetingLobby} />,
+      <MenuItem icon="clock-o" label="Start Retro Meeting" onClick={goToRetroLobby} />
+    );
+    return listItems;
+  };
+
+  const buttonToggle = (
+    <Button
+      buttonSize="small"
+      buttonStyle="primary"
+      colorPalette="warm"
+      icon="chevron-down"
+      iconPlacement="right"
+      isBlock
+      label="Start Meeting"
+    />
+  );
+
   return (
     <ButtonGroup>
-      <ButtonBlock>
+      {__RELEASE_FLAGS__.retro ?
+        <Menu
```

Comment:
Ah shoot, i need to flag the Menu component as deprecated in favor of the more accessible, cleaner MenuWithItems. I'll go ahead & switch components